openpower/package/hostboot/p9Patches/hostboot-0004-remove-sbfw-check-hbDistribute.patch

```diff
@@ -0,0 +1,40 @@
+From 52929548d943e0387fabb70d658905120cad7cd8 Mon Sep 17 00:00:00 2001
```

Comment:
Not sure if I understand what you are saying - but RTC 160442 is open and I do have this change up for review in local hostboot.